src/main/ApplicationImpl.cpp

```diff
@@ -142,6 +143,7 @@ ApplicationImpl::initialize(bool createNewDB)
     ConservationOfLumens::registerInvariant(*this);
     LedgerEntryIsValid::registerInvariant(*this);
     LiabilitiesMatchOffers::registerInvariant(*this);
+    OrderBookIsNotCrossed::registerInvariant(*this);
```

Comment:
I don't think we want to allow people to enable this invariant in production (at least right now :) )